WPMedia/fusion-news-theme-blocks
# THEMES-1589: Subs phrases added for all locales

## Description

This PR ensures that the new phrases added for the Subs blocks are in all locale files, even if they're not translated yet.

## Jira Ticket

- [THEMES-1589](https://arcpublishing.atlassian.net/browse/THEMES-1589)

## Acceptance Criteria

1. New Subs block phrases are added to Lokalise.
2. All locale files in arc-themes-blocks are updated with the new phrases.

## Test Steps

1. Verify the new phrases have entries for all languages in `subscriptions-block/intl.json`.
2. Verify that the new phrases are in the language `json` files in the `locale` directory.
3. Verify the new phrases have been entered into the Lokalise project: https://app.lokalise.com/project/704194126082f2d6689a09.40341174/?view=multi

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1589]: https://arcpublishing.atlassian.net/browse/THEMES-1589?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
